### DISCUSSION TITLE
Field edge buffer

### DISCUSSION THREAD

**Discussion by crcarmichael** [Courseplay/courseplay#7122] [Ideas]
It would be great if we could add a "field edge buffer" when generating courses.  Up to 2m inside the field for fields that have trees and other obstacles close to the edge.

**Comment by Tensuko** [comment]
2m would be to much, as we mostly just touch the trees.
The 1st Headland got no overlap, like the 2nd and more.
What I also can suggest you is to use the ingame paint tool, paint a bit grass at the edge where the tree is, scan Field Border and save it as User Field. Very simple, works right now :)

But we think of this anyway.

**Reply by crcarmichael** [reply]
All of my fields are perfect... it's the contract fields that cause problems. 😉

**Reply by Tensuko** [reply]
Can do the same with any Field.

**Reply by crcarmichael** [reply]
> Can do the same with any Field.

OMG. I have hundreds of hours in this game and I never realized you can landscape land you don't own. 🤦‍♂️

**Reply by Tensuko** [reply]
> 
> 
> > Can do the same with any Field.
> 
> OMG. I have hundreds of hours in this game and I never realized you can landscape land you don't own. 🤦‍♂️

Forgot about this... lol if you can rly do this, I can't haha. Anyway as I said we think about a solution for this...

**Reply by crcarmichael** [reply]
> > > Can do the same with any Field.
> > 
> > 
> > OMG. I have hundreds of hours in this game and I never realized you can landscape land you don't own. 🤦‍♂️
> 
> Forgot about this... lol if you can rly do this, I can't haha. Anyway as I said we think about a solution for this...

Ok, I'm not crazy!  I think it must be just this map (Bucks County) that allows it.  I think most maps don't allow it.

**Reply by pvaiko** [reply]
Created #7141 this should be easy to implement.
